diesel/src/query_dsl/mod.rs

```diff
@@ -752,6 +870,44 @@ pub trait RunQueryDsl<Conn>: Sized {
     /// `Err(NotFound)` will be returned if the query affected 0 rows. You can
     /// call `.optional()` on the result of this if the command was optional to
     /// get back a `Result<Option<U>>`
+    ///
+    /// When this method is called on an insert, update, or delete statement,
+    /// it will implicitly add a `RETURNING *` to the query,
+    /// unless a returning clause was already specified.
+    ///
+    /// # Example
+    ///
+    /// ```rust
+    /// # #[macro_use] extern crate diesel;
+    /// # include!("../doctest_setup.rs");
+    /// #
+    /// # fn main() {
+    /// #     run_test();
+    /// # }
+    /// #
+    /// # #[cfg(feature = "postgres")]
```

Comment:
There should be a visible sign that this code only works with postgres.

Reply:
I don't think we need to state "sqlite and mysql don't support returning clauses" every time we say "RETURNING" in our docs. Especially since trying to run this code on SQLite will give you "`Sqlite: SupportsReturningClause` is not satisfied"